spec/calendar.html

```diff
@@ -204,7 +204,8 @@ <h1>CalendarDayOfWeek ( _calendar_, _dateLike_ )</h1>
       </p>
       <emu-alg>
         1. Assert: Type(_calendar_) is Object.
-        1. Return ? Invoke(_calendar_, *"dayOfWeek"*, « _dateLike_ »).
+        1. Let _result_ be ? Invoke(_calendar_, *"dayOfWeek"*, « _dateLike_ »).
+        1. Return ? ToPositiveInteger(_result_).
```

Comment:
I realized that this changes the return type of these operations, so we need to add an extra 𝔽() around the places where we return this value to JS.